app/services/analytics_events.rb

```diff
@@ -2652,8 +2777,40 @@ def idv_in_person_proofing_redo_state_id_submitted(
     )
   end
 
-  def idv_in_person_proofing_residential_address_submitted(**extra)
-    track_event('IdV: in person proofing residential address submitted', **extra)
+  # @param [Boolean] success Whether form validation was successful
```

Comment:
NIt- I did not look at all events but it would be nice if the order matched  ie: 3 arg in notes is error_details but is flow_path in args below

Reply:
I tried to add some consistency here while balancing diff size. I think we'll want to make a pass at standardizing order & descriptions once we've got everything fully documented

Reply:
Make sense to me- resolving